.gitignore

```diff
@@ -64,3 +64,6 @@ node_modules
 *.swp
 
 *.tsbuildinfo
+
+# VS Code settings
+.vscode/*
```

Comment:
I personally disagree on having this kind of gitignores on repos, it doesn't scale. You can have a global gitignore on your setup.